recipes/godot-cpp/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "cci.3.2-20200130":
```

Comment:
The version is very unique :worried:

Reply:
It matches the one from `godot_headers`... I agree it is a bit unique, maybe we can figure out something better in the future.